main.go

```diff
@@ -27,13 +35,17 @@ func timing(step string, f func()) {
 
 var Version string
 
+//GotchaCAPath is a constant containing the path to the .gotcha folder which contains the CA/certs/keys gotcha will use for TLS
+var GotchaCAPath = os.Getenv("HOME") + "/.gotcha"
```

Comment:
This var isn't used outside of loadOrGenerateCA(), so let's just make it a local var to that function.